charts/db-operator/crds/kci.rocks_databases.yaml

```diff
@@ -71,6 +71,14 @@ spec:
                 - cron
                 - enable
                 type: object
+              connectionStringTemplate:
```

Comment:
can you add it also in doc https://github.com/kloeckner-i/db-operator/blob/master/docs/creatingdatabases.md ?

Reply:
Done